src/test/java/org/testeditor/web/xtext/index/XtextIndexServiceTest.xtend

```diff
@@ -0,0 +1,70 @@
+package org.testeditor.web.xtext.index
+
+import com.google.inject.Injector
+import org.eclipse.emf.common.util.URI
+import org.eclipse.emf.ecore.EPackage
+import org.eclipse.emf.ecore.EReference
+import org.eclipse.emf.ecore.util.EcoreUtil
+import org.eclipse.xtext.XtextPackage
+import org.eclipse.xtext.XtextStandaloneSetup
+import org.eclipse.xtext.mwe.ResourceDescriptionsProvider
+import org.eclipse.xtext.naming.QualifiedName
+import org.eclipse.xtext.resource.XtextResourceSet
+import org.junit.Before
+import org.junit.Test
+
+import static org.assertj.core.api.Assertions.assertThat
+
+class XtextIndexServiceTest {
+
+	var Injector injector
+
+	@Before
+	def void setupXtextInjector() {
+		injector = new XtextStandaloneSetup().createInjectorAndDoEMFRegistration
+	}
+
+	@Test
+	def void shouldLoadResourceIntoIndex() {
+		// given
+		val resourceSet = injector.getInstance(XtextResourceSet)
+		resourceSet.getResource(URI.createFileURI("src/test/resources/index/MyDsl.xtext"), true)
+		val expectedName = QualifiedName.create("org", "xtext", "example", "mydsl", "MyDsl")
```

Comment:
I like to see direct the expected result of the test. When we need to create the expectation the test is difficult to understand. Why not hard code the expectation?